dm/master/server.go

```diff
@@ -1441,7 +1442,7 @@ func (s *Server) resolveDDLLock(ctx context.Context, lockID string, replaceOwner
 		// owner execute DDL fail, do not continue
 		return []*pb.CommonWorkerResponse{
 			ownerResp,
-		}, errors.Errorf("owner %s ExecuteDDL fail", owner)
+		}, terror.ErrMasterOwnerExecDDL.Generate(owner)
```

Comment:
Is L1436 need to be updated with code/class/level (and other places with `errors.ErrorStack`)?

Reply:
`err` used in L1436 is returned from `cli.SendRequest`, which has already inclued error code/class/level.